src/System.Xml.XPath.XDocument.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.22310.1
```

Comment:
Ah sorry, I thought the change affected `MinimumVisualStudioVersion` as well which would've broken VS2013. I don't care then.